.github/workflow-templates/bridge-e2e/action.yml

```diff
@@ -67,6 +114,11 @@ runs:
       id: check_yarn
       run: |
         yarn --version
+    - name: Use Node.js 22.x
```

Comment:
Probably we need to fix it in the other places, but since we added the `.nvmrc` recently, probably it's a good idea to read the version from there, in this case, we can update all the CI in 1 place. The example:
```
      - name: Read .nvmrc
        id: nvmrc
        run: echo "##[set-output name=NODE_VERSION;]$(cat .nvmrc)"
      
      - name: 'Set up node'
        uses: actions/setup-node@v2
        with:
          node-version: "${{ steps.nvmrc.outputs.NODE_VERSION }}"
```

Reply:
parameterized this now and given it a default val. the nvmrc step doesnt exist in the `merge-when-ready.yml` so that will need to be added down the road.